docs/application/dotnet/tutorials/test/testing-your-app-on-gear.md

```diff
@@ -0,0 +1,72 @@
+# Test Tizen .NET Application on a Gear device
+
+To deploy Tizen .NET Application to Samsung Gallaxy app store, you need to test it on Samsung Gear devices.
+As Samsung Gear doesn't have a USB port, Tizen provides a way to do it over Wi-Fi.
```

Comment:
Since, Samsung Gear does not have a USB port, Tizen provides a support to publish the app to the app store over Wi-Fi.

Please verify whether the technical meaning is altered.

Reply:
done